docs/source/ledger-nano.rst

```diff
@@ -0,0 +1,263 @@
+.. _ledger-nano:
+
+Manage XNS via your Ledger Nano hardware wallet
+===============================================
+
+`Ledger Nano S <https://shop.ledger.com/products/ledger-nano-s>`_ and `Nano X <https://shop.ledger.com/products/ledger-nano-x>`_ are hardware wallets built by `Ledger <https://www.ledger.com/>`_.
+
+A hardware wallet is a separate device that stores the private keys for your crypto assets. The keys never leave the device and remain secure even if the device is connected to a compromised computer.
+
+The Insolar application for Ledger Nano S and X keeps your Insolar private key on the device and lets you manage XNS coins via the `Insolar Wallet <https://wallet.insolar.io>`_.
+
+The application is developed by the Insolar team. To request technical support, send an email to support@insolar.io.
+
+Prerequisites
+-------------
+
+To use the Insolar application on your Ledger Nano S or X hardware wallet, make sure you install and set up the following:
+
+#. `Install Google Chrome <https://www.google.com/chrome/>`_. It's the only supported browser for now.
+#. `Install the Ledger Live application <https://support.ledger.com/hc/en-us/articles/360006395553/>`_. It allows you to manage applications on your device.
+#. If you don't yet have an Insolar Wallet, first set up your `Nano S <https://support.ledger.com/hc/en-us/articles/360000613793>`_ or `Nano X <https://support.ledger.com/hc/en-us/articles/360018784134>`_. Then see if you need to install a firmware update in the next step.
+
+   .. important::
+
+     During the setup, you are required to choose a PIN and write down the recovery phrase for the device. Without the PIN you will be unable to unlock the device and without the recovery phrase you will be unable restore access to your wallet.
+
+   Otherwise, if you do have an Insolar Wallet, initialize your device with the existing backup phrase:
+
+   #. Follow the official instructions for `Nano S <https://support.ledger.com/hc/en-us/articles/360005434914>`_ or `Nano X <https://support.ledger.com/hc/en-us/articles/360015132494>`_ to initialize the wallet with the phrase.
+   #. At **step 3** of the official instruction, choose the 18-word length and enter your Insolar Wallet's backup phrase.
```

Comment:
_Still don't get what the "18-word length"  thing is. Feels like there is a word missing_